frontend/src/components/Common/ReportDialog/reportDoc.tsx

```diff
@@ -89,7 +89,7 @@ const ReportDoc = memo(
     columns,
   }: ReportDocProps) => {
     if (mapImage === null) {
-      return <></>;
+      return <Document />;
```

Comment:
I think this was causing an error before. See https://github.com/WFP-VAM/prism-app/pull/624
Ask @echaidemenos for more info if needed

Reply:
Yes you are right @ericboucher. This was causing an error but adding the `<></>` component was causing other console errors. I've made a fix that removes the if statement at the root of the report doc component and prevents the auto downloading of the pdf when mounted and not opened. I'll make some tests in different browsers and windows firefox that we had this issue before to see that everything is going as expected